### ISSUE TITLE
Change UserBaseHandler.get_current_user to return an ID not the entire ORM entry

### ISSUE THREAD

**Issue by linted** [linted/HardwareCheckout#124]
This change will fix some of our overuse of DB connections.

**Comment by 5hu5ky** [comment]
looking at this - not seeing any DB usage in this function... Only getting user cookie (lines 23-24):
```
            user_cookie = int(self.get_secure_cookie('user'))
            return user_cookie
```

**Comment by linted** [comment]
I'm sorry this ticket is closed with #125
